test/common.py

```diff
@@ -933,11 +937,10 @@ def require_jspi(self):
 
     exp_args = ['--experimental-wasm-stack-switching', '--experimental-wasm-type-reflection']
     nodejs = self.get_nodejs()
+    print('NODEJS is ' + str(nodejs))
```

Comment:
Debugging

Reply:
yeah this is very weird, i.e. whether the variables that are node are actually lists or strings. Thanks duck-typing!